packages/client/bin/cli.ts

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env client
+#!/usr/bin/env node
```

Comment:
This was actually breaking the installation completely (respectively the client couldn't be used with the `ethereumjs` command. 😜